src/crypto/RawKeySessionKeystore.cpp

```diff
@@ -19,10 +19,21 @@
 
 #include <lib/support/BufferReader.h>
 
+#include <cstdint>
+
 namespace chip {
 namespace Crypto {
 
-using HKDF_sha_crypto = HKDF_sha;
+// The underlying representation of the HKDF key handle
+struct RawHkdfKeyHandle
+{
+    ByteSpan Span() const { return ByteSpan(data, size); }
+
+    static constexpr size_t kMaxDataSize = std::min(CHIP_CONFIG_HKDF_KEY_HANDLE_CONTEXT_SIZE - 1, UINT8_MAX);
```

Comment:
Why the `- 1` ?

Reply:
Because the size itself must fit in the context.

Reply:
Added comment on that.